src/seedu/addressbook/ui/TextUi.java

```diff
@@ -169,4 +164,38 @@ private static String getIndexedListItem(int visibleIndex, String listItem) {
         return String.format(MESSAGE_INDEXED_LIST_ITEM, visibleIndex, listItem);
     }
 
+    /**
+     * formatter class to format text for display
+     */
+    static class Formatter{
+
+        /**
+         *
+         * @param displayType format text either in "Enter command" format or messages to user
+         * @param textToDisplay text that want to be formatted
+         * @return formatted text
+         */
+        public String formatTextForDisplay(int displayType, String textToDisplay){
```

Comment:
Instead of asking the caller to provide the displayType, you could have created separate public methods (eg. formatEnterCommand() and formatDisplayMessages()) that called this method with the appropriate displayType.

Reply:
okay, thanks for the suggestion